src/config/dynamic-theme-fixes.config

```diff
@@ -3551,6 +3551,20 @@ CSS
 
 ================================
 
+downdetector.*
+allestoringen.*
+xn--allestrungen-9ib.*
+
+CSS
+.img-fluid {
+    filter: invert(0.9) hue-rotate(180deg) brightness(0.8) !important;
+}
+#map-card {
```

Comment:
I don't really like the usage of custom filters as they can go out of sync by user's settings. 
Also `.mapboxgl-canvas` is another one to be included for INVERT.

Reply:
Thanks.  Which user settings can mess up custom filters?

Initially, I thought I agreed with you that applying a real complex custom filter could possibly create a big issue for some people, but after a little more thought, I'm haven't come up with real-world situations where this would be the case.  IIRC, the resultant image should always be the same if DR is working correctly.

The only issue I can think of off-hand is maybe when someone is, for some reason, using Dark Reader to force a light dynamic theme.  I'm not sure how many people actually do that... do you have any idea?  Even then, I'm not sure if it's an issue.

Regarding this specific filter (no longer speaking in generalities), just inverting, rotating the hue, and lowering the brightness a touch I think would work for just about everyone (if not everyone), especially the main target audience (people who want a dark theme without excessive contrast).  I'm interested in learning about cases where a relatively simple filter like this will cause an issue, though.  I've used them in code outside of DR, and I want to ensure there isn't a case I'm missing.

Reply:
> Thanks. Which user settings can mess up custom filters?

Rather meant that certain setting the user touch brightness/contrast/sepia/grayscale, will be reflected on the page and also be reflected into the `INVERT` section. However with hardcoded filter's like these, they won't. And just stay static while the page is changing.

> I'm haven't come up with real-world situations where this would be the case. IIRC, the resultant image should always be the same if DR is working correctly.

I can think of one. Let's take the canvas as a example, as I have on every site with a few exceptions a very high contrast(141%), due to partially how my display is not really reflecting contrast as it should so it's more in balanced. I really notice hard coded values:

if you use the `INVERT` section
![image](https://user-images.githubusercontent.com/25481501/122729018-3c98a300-d268-11eb-925f-16770e066c69.png)

using the hardcoded value:
![image](https://user-images.githubusercontent.com/25481501/122729130-5afe9e80-d268-11eb-8c6a-3647f348dbd0.png)


^ I've just noticed it doesn't have filter/hue-rotate which I really dislike especially in this situation.

So using the hardcoded value & `filter(1) hue-rotate(180deg)`
![image](https://user-images.githubusercontent.com/25481501/122729279-841f2f00-d268-11eb-8667-5ea6dc4b8bad.png)

It's notice-able that's out of sync with the site, and every person encountering this will probably complain and a few of them will open up a bug report of this. 

> I'm not sure how many people actually do that... do you have any idea? Even then, I'm not sure if it's an issue.

Not really a lot, that I can think of... only users that have automation settings and don't disable dark reader at day but rather use the light theme.

> just inverting, rotating the hue, and lowering the brightness a touch I think would work for just about everyone (if not everyone),

Correct, should be good to go... however for people with custom filter settings, they will be notice-able that it isn't a correct fix. 

> especially the main target audience (people who want a dark theme without excessive contrast).

Yeah, I thought that was the main audience too, but I've failed to realize that a good amount of users also use it for other reasons, See #6032

> I've used them in code outside of DR, and I want to ensure there isn't a case I'm missing.

If something doesn't allow variables to play in and simply force such hardcoded value and you know that the site won't be changing, that is good and shouldn't be a problem. However I really try with these fixes to not break user experience, because they are out-of-sync. Il allow certain hardcoded values, but really rare edge cases(e.g. edges of QR codes so they will be recognized).

Reply:
Thank you for all your feedback and insight.

Using this site as an example, how to you recommend handling those logos?  When I use the DR dynamic dark mode without any custom settings, some of them are virtually invisible because they are too dark, while others are too bright.